lambda_2/lambda_function.py

```diff
@@ -0,0 +1,52 @@
+import json
+import csv
+import boto3
+import os
+import pandas as pd
+import awswrangler as wr
+from datetime import datetime
+
+def handler(event, context):
+    source_bucket_name = os.environ['BUCKET_NAME']
+    file_key = event['Records'][0]['s3']['object']['key']
+
+    output_file_name = 'processed-csv-file'
+
+    s3 = boto3.resource('s3')
+    source_bucket = s3.Bucket(source_bucket_name)
+
+    json_file = source_bucket.Object(file_key).get()['Body'].read().decode('utf-8')
+    data = json.loads(json_file)
+
+    flattened_data = process_json(data)
+
+    csvdf = pd.DataFrame(flattened_data)
+
+    csv_file_path = f's3://{event["bucketName"]}/processed-csv-file.csv'
+    c(df=csvdf, path=csv_file_path, index=False)
```

Comment:
Is this meant to be a wrangler command to_csv?

Reply:
yep, changed it